submissions/busolaAdedoyin/css/contact.css

```diff
@@ -168,12 +173,20 @@
  .info-logo li{
       margin:10px;
  }
- .inputs input[type=submit] {
+/* .inputs input[type=submit] {
```

Comment:
@elizabethbusola Do you want to revisit this comment block or you are not using it anymore?

Reply:
I have deleted it and committed again. I dont need it anymore @ToluAkin